content/fr/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -0,0 +1,111 @@
+
+_EndpointSlices_ offrent une simple methode pour suivre les endpoints d'un réseau au sein d'un cluster de Kubernetes. Ils offrent une alternative plus evolutive et extensible aux Endpoints.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Resource pour EndpointSlice {#endpointslice-resource}
+
+Dans Kubernetes, un EndpointSlice contient des reférences à un ensemble de reseau 
+endpoints. Le controleur d'EndpointSlice crée automatiquement des EndpointSlices pour un Kubernetes Service quand un {{< glossary_tooltip text="selector" term_id="selector" >}} est spécifié. Ces EnpointSlices vont inclure des references a n'importe quelle Pods qui correspond aux Service selector. EndpointSlices groupent ensemble les endpoints d'un reseau par combinaisons uniques de Services et de Ports.
+
+Par example, voici un échantillon d'une resource EndpointSlice pour le Kubernetes Service `example`.
+
+```yaml
+apiVersion: discovery.k8s.io/v1beta1
+kind: EndpointSlice
+metadata:
+  name: example-abc
+  labels:
+    kubernetes.io/service-name: example
+addressType: IPv4
+ports:
+  - name: http
+    protocol: TCP
+    port: 80
+endpoints:
+  - addresses:
+      - "10.1.2.3"
+    conditions:
+      ready: true
+    hostname: pod-1
+    topology:
+      kubernetes.io/hostname: node-1
+      topology.kubernetes.io/zone: us-west2-a
+```
+
+EndpointSlices geré par le controleur d'EndpointSlice n'auront, par défaut, pas plus de 100 endpoints chacun. En dessous de cette échelle, EndpointSlices devrait mapper 1:1 les Endpoints et les Service et devrait avoir une performance similaire.
+
+EnpointpointSlices peuvent agir en tant que source de vérité pour kube-proxy quand it s'agit du routage d'un trafic interne. Lorsqu'ils sont activés, ils devront une amélioration de performance pour les services qui ont une grand quantité d'endpoints.
+
+### Types d'addresses
+
+EndpointSlices supporte trois type d'addresses:
+
+* IPv4
+* IPv6
+* FQDN (Fully Qualified Domain Name) - [serveur entièrement nommé]
+
+### Topologie
+
+Chaque endpoint dans un EnpointSlice peut contenir des informations de topologie pertinentes. 
+Ceci est utilisé pour indiqué où se trouve un endpoint, qui contient les informations sur le Noeud, zone et region correspondant. Lorsque les valeurs sont disponibles, les etiquette de Topologies suivantes seront définies par le contrôleur EndpointSlice:
+
+* `kubernetes.io/hostname` - Nom du Noeud sur lequel l'endpoint se situe.
+* `topology.kubernetes.io/zone` - Zone dans laquelle l'endpoint se situe.
+* `topology.kubernetes.io/region` - Region dans laquelle l'endpoint se situe.
+
+Le contrôleur EndpointSlice surveille les Services et les Pods pour assurer que les correspondantes EndpointSlices sont mis-à-jour. Le contrôleur gèrera les EndpointSlices pour tout les Services qui ont un selecteur - [reference: {{< glossary_tooltip text="selector" term_id="selector" >}}] - specifié. Celles-ci representeront les IPs des Pods qui correspond au selecteur.
```

Comment:
`text="selector"`? J'écrivais `text="selecteur"`.

Reply:
Un autre cas ou j'étais pas si je devais traduir ou pas. Je traduirai le reste.

Reply:
Crée une erreur `"selecteur" is not a valid glossary term_id, see ./docs/reference/glossary/* for a full list`.

Reply:
Try this: `{{< glossary_tooltip text="selecteur" term_id="selector" >}}`

Reply:
made the changes seems to work fine